velox/type/Timestamp.h

```diff
@@ -30,11 +31,12 @@ class time_zone;
 
 namespace facebook::velox {
 
-struct Timestamp {
+struct Timestamp : public ISerializable {
```

Comment:
We need to avoid create vtable for such a low level class.  Can we do the serialization of this class without this inheritance?

Reply:
But in order to use `ISerializable::deserialize<Timestamp>(obj["lower"])` it has to inherit `ISerializable` right?

Reply:
Or we can just fallback to previous version - do the serialization on the `TimestampRange` level:
```C++
	
folly::dynamic TimestampRange::serialize() const {
  auto obj = Filter::serializeBase("TimestampRange");
  obj["lowerSeconds"] = lower_.getSeconds();
  obj["lowerNanos"] = lower_.getNanos();
  obj["upperSeconds"] = upper_.getSeconds();
  obj["upperNanos"] = upper_.getNanos();
  return obj;
}
```

Reply:
What would be the `obj` here? And how are we gonna serialize `Timestamp` without inheriting `Serializable` and registering it?

Reply:
The obj is the `folly::dynamic` we received.  The method look up the `create` method on the type and call it, not registration needed.